pydantic/validators.py

```diff
@@ -24,6 +24,8 @@
 )
 from uuid import UUID
 
+from pydantic.typing import get_class
```

Comment:
```suggestion
from .typing import get_class
```

and you'll need to run `make format` again.